### ISSUE TITLE
notification drawer header text not flexible

### ISSUE THREAD

**Issue by jschuler** [patternfly/patternfly-react#4651]
If you want to see the unread count text in the Notification header, you need to pass `count` and an optional `unreadText` prop to the NotificationDrawerHeader, it will render it as 
`${count} ${unreadText}` which is not very i18n friendly as it strictly limits you to having a number followed by a space and a string.

**Comment by mturley** [comment]
We could maybe add an optional prop `formatHeaderText` which is a function `(count: number, unreadText: string) => string` and it could default to returning `${count} ${unreadText}` but could be overridden?

**Comment by DaoDaoNoCode** [comment]
I agree with @mturley , but could we make it even more flexible? Can we simply not pass any props like `count` and `unreadText` into the `formatHeaderText` function but just `() => string`? Because we can also use `count` and `unreadText` here if we want to customize the information. Also, it will show `${count} ${unreadText}` when we don't set this function and only pass `count` and `unreadText` into the `NotificationDrawerHeader`.
